### ISSUE TITLE
Unable to add somfy screens to TaHoma  app

### ISSUE THREAD

**Issue by jellehvanbuuren** [AdyRock/com.somfy.tahoma#87]
I have installed the TaHoma app version v4.0.13, but it does not find any connected devices.
In the TaHoma app itself I do find these io devices and can control them without problems.
I'm using the beta version of Homey.

I created a log file to review. the key is 5eb3b117-12c7-4788-867b-727fd24dc5ae

Thanks
Jelle

**Comment by AdyRock** [comment]
Unfortunately, those logs don't provide much information:
```
    User Message:
    Ik vind geen aangesloten devices.
Ik heb 3 io screens geinstallers in de tahoma app en die werken prima.

    stdout:
    n/a

    stderr:
    ue, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
syncLoop, Logged in = true, Local = false, Old Sync State = false
Fetching Events, Complete
```

All I can suggest is to try different devices to see if yours appear under a different type.

The good news is it has let you log in OK.

**Comment by AdyRock** [comment]
I have published a new test version "4.0.15" where I am logging the device discovery to this.homey.log to see if it shows up in the diagnostic report.
Could you install the test version, try adding a device and then send a new Diagnostic report.

**Comment by jellehvanbuuren** [comment]
Hi Adrien.  Thx for the quick action, but I found the devices.  I found them under the (confusing) name Pergola.
